docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Service Programming Model
+type: Details
+---
+# Service Programming Model
+
+A microservice deployed in Kyma can be configured to receive events from the event bus. This is achieved by creating a [subscription custom resource](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/eventing-subscription.crd.yaml). Refer to [consume events section](https://github.com/kyma-project/kyma/blob/master/docs/event-bus/docs/011-details-event-flow-requirements.md#consume-events) for various configuration options.
+
+## Event Delivery
+
+```ascii
```

Comment:
Please follow the guidelines when creating diagrams. The guidelines for diagrams are here:
https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/diagrams.md

Use draw.io and have fun :) 
After the diagram is good, I'll check the documentation.

Reply:
@klaudiagrz Thanks for the tip. please check now!